cloud/aws/common/index.go

```diff
@@ -32,27 +32,31 @@ type Topic struct {
 
 // ResourceIndex - The resource index for a nitric stack
 type ResourceIndex struct {
-	Buckets     map[string]AwsResourceArn `json:"buckets"`
-	Topics      map[string]Topic          `json:"topics"`
-	KvStores    map[string]AwsResourceArn `json:"kvStores"`
-	Queues      map[string]AwsResourceArn `json:"queues"`
-	Secrets     map[string]AwsResourceArn `json:"secrets"`
-	Apis        map[string]ApiGateway     `json:"apis"`
-	HttpProxies map[string]ApiGateway     `json:"httpProxies"`
-	Websockets  map[string]ApiGateway     `json:"websockets"`
-	Schedules   map[string]AwsResourceArn `json:"schedules"`
+	Buckets        map[string]AwsResourceArn `json:"buckets"`
+	Topics         map[string]Topic          `json:"topics"`
+	KvStores       map[string]AwsResourceArn `json:"kvStores"`
+	Queues         map[string]AwsResourceArn `json:"queues"`
+	Secrets        map[string]AwsResourceArn `json:"secrets"`
+	Apis           map[string]ApiGateway     `json:"apis"`
+	HttpProxies    map[string]ApiGateway     `json:"httpProxies"`
+	Websockets     map[string]ApiGateway     `json:"websockets"`
+	Schedules      map[string]AwsResourceArn `json:"schedules"`
+	Distributions  map[string]AwsResourceArn `json:"distributions"`
+	WebsiteBuckets map[string]AwsResourceArn `json:"websiteBuckets"`
```

Comment:
Are these being referenced at runtime? Do they need to be included in the index for SSM?

Reply:
No they are not referenced at runtime

Reply:
can probably take these out of the resource index for now then, generally only used for things we need to access properties of at runtime

Reply:
Ok I will remove them